README.md

```diff
@@ -33,9 +33,10 @@ lines:
 }
 ```
 
-After that, run the following command in the root directory of Nextcloud on the server
+After that, run the following commands in the root directory of Nextcloud on the web server
 (if needed, replace `www-data` with the actual user which is used by the webserver):
 
+    sudo -u www-data php occ maintenance:mimetype:update-js
```

Comment:
remove `update-js` and `files:scan` in favor of `sudo -u www-data php occ maintenance:mimetype:update-db --repair-filecache` as stated in my comment